apps/dashboard/forms.py

```diff
@@ -273,3 +276,42 @@ class ProfileForm(forms.ModelForm):
     class Meta:
         model = User
         fields = ['username', ]
+
+
+class AddModeratorForm(forms.ModelForm):
+    add_moderators = CommaSeparatedEmailField(label=_('Add moderator via '
+                                                      'email'))
+
+    class Meta:
+        model = project_models.Project
+        fields = ('moderators',)
+
+    def __init__(self, *args, **kwargs):
+        self.request = kwargs.pop('request', None)
+        super().__init__(*args, **kwargs)
+
+    def clean_add_moderators(self):
+        users = []
+        missing = []
+        for email in self.cleaned_data['add_moderators']:
+            try:
+                user = User.objects.get(email__exact=email)
+                users.append(user)
+            except ObjectDoesNotExist:
+                missing.append(email)
+
+        if missing:
+            messages.error(
+                self.request,
+                _('Following e-mails are not registered: ') + ', '.join(
+                    missing)
+            )
+
+        return users
```

Comment:
That's built-in, see 5th bullet point: https://docs.djangoproject.com/en/1.10/ref/forms/validation/